macaw-loader/src/Data/Macaw/BinaryLoader.hs

```diff
@@ -82,9 +91,10 @@ class (MM.MemWidth (MR.ArchAddrWidth arch)) =>
   -- particular, an ELF file usually specifies an entryPoint address
   -- that is also an address in the symbol table, both of which are in
   -- the data returned by this function.
-  entryPoints :: (X.MonadThrow m) =>
-                 LoadedBinary arch binFmt
-              -> m [MM.ArchSegmentOff arch]
+  entryPoints ::
+    (X.MonadThrow m) =>
+    LoadedBinary arch binFmt ->
+    m [MM.ArchSegmentOff arch]
```

Comment:
(Caveat: I do not generally endorse the use of a linter for common style).  Are these changes being driven by some form of linter or is  there another reason for changing the existing style?